client/app/components/Accordion/index.jsx

```diff
@@ -30,7 +30,7 @@ export const Accordion = ({
   const [open, setOpen] = useState(!!openProp);
 
   const displayContent = () => (
-    <div className={`${open ? 'accordionContent' : css.accordionClose}`}>
+    <div className={`${open ? 'accordionContent' : css.accordionClose}`} role="list">
```

Comment:
So we shouldn't be using this list role on this accordion. It would be better to identify this element by an aria-label.